proposals/lip-0049.md

```diff
@@ -572,7 +576,8 @@ def verify(ccu: Transaction, ccm: CCM) -> None:
 ##### Execution
 
 ```python
-def execute(ccu: Transaction, ccm: CCM) -> None:
+def execute(ccu: CCU, ccm: CCM) -> None:
+    ccmParams = decode(registrationCCMParamsSchema, ccm.params)
```

Comment:
it seems that we don't need to decode the params in this function

Reply:
I removed this line.